db/migrate/20200824112700_facilities_teleconsultation_medical_officers.rb

```diff
@@ -0,0 +1,10 @@
+class FacilitiesTeleconsultationMedicalOfficers < ActiveRecord::Migration[5.2]
+  def change
+    create_join_table :facilities,
+      :users,
+      column_options: {type: :uuid},
+      table_name: "facilities_teleconsultation_medical_officers" do |t|
+      t.index [:facility_id, :user_id], name: :index_facilities_teleconsult_mos_on_facility_id_and_user_id
+    end
+  end
+end
```

Comment:
Is there any merit in setting up a first-class join table like "TeleconsultationMedicalOfficer" that joins facilities and users? Do we foresee the need for more teleconsultation logic in the near future that would live there?

Reply:
PS Either way, we should set up a model to access this join table. Otherwise, it becomes awkward to ask questions like "Show me the list of teleconsult MOs".

Reply:
I'd like to not call the join model `TeleconsultationMedicalOfficer` since it doesn't represent a real MO, it's just a relation underneath. wdyt about 21ec1bb? This has the advantage that `TeleconsultationMedicalOfficer.all` _are_ users. One doesn't have to do:
```
t = TeleconsultationMedicalOfficers.first
u = t.user // t should already be a user
```

We could add _a_ model to back the join table if we need it. The join table doesn't do anything interesting though currently and we could always add it later, I think.

Reply:
I like this 👍  Let's do it!